lib/pcr.c

```diff
@@ -211,6 +213,45 @@ bool pcr_print_pcr_struct(TPML_PCR_SELECTION *pcrSelect, tpm2_pcrs *pcrs) {
 }
 
 
+bool pcr_print_pcr_selection(TPML_PCR_SELECTION *pcrSelect) {
+    UINT32 i;
+    int j;
+    bool first;
+    const char* halgstr;
```

Comment:
del halgstr, first, j and i from here. Declare closes to use per coding standards.

Reply:
ack